test/com/collections/MethodsTest.java

```diff
@@ -139,31 +140,52 @@ void canBePalindrome() {
         assertFalse(methods.canBePalindrome("kajakk"));
     }
 
-    @Test
-    void sumTwoIntArrays() {
+    @Test(dataProvider = "sumOfArrays1DataProvider")
+    void sumTwoIntArrays1(int [] array1, int [] array2, int [] sumA1A2 ) {
+        assertTrue(compareTwoIntArrays(methods.sumTwoIntArrays1(array2, array1), sumA1A2));
+    }
+
+    @DataProvider(name = "sumOfArrays1DataProvider")
 
-        //1 Case
-        int[] array1 = {1, 2, 3};
-        int[] array2 = {7, 9, 9};
-        int[] sumA1andA2 = {1, 1, 2, 2};
-        int[] methodSum1 = methods.sumTwoIntArrays(array1, array2);
+    public Object[][] getData1() {
+        int[] array1 = {2, 4, 6};
+        int[] array2 = {1, 2};
+        int[] sumA1A2 = {2, 5, 8};
 
-        assertTrue(compareTwoIntArrays(sumA1andA2,methodSum1));
+        int[] array3 = {9, 8, 7, 6, 2, 5, 9, 8, 6, 2, 5, 9, 8, 6, 5, 2, 3};
+        int[] array4 = {9, 8, 7, 8, 9, 7, 8, 7, 2, 3, 5};
+        int[] sumA3A4 = {9, 8, 7, 6, 2, 6, 9, 7, 4, 1, 5, 7, 7, 3, 7, 5, 8};
+
+        int[] array5 = {1, 2, 3};
+        int[] array6 = {1, 2, 3};
+        int[] sumA5A6 = {2, 4, 6};
+
+
+        return new int[][][]{{array1, array2, sumA1A2},{array3,array4,sumA3A4},{array5,array6,sumA5A6}};
+    }
+
+    @Test(dataProvider = "sumOfArraysDataProvider")
```

Comment:
I think same data provider can be used here instead of two. Please also name both **methods.sumTwoIntArrays** in more appropriate way (instead of adding **1** in name).